Readme.md

```diff
@@ -30,51 +34,51 @@ If you already have a user account in your Azure Active Directory tenant, you ca
-7. Enter a friendly name for the application, for example "TodoListWebApp_MT", select "Web Application and/or Web API", and click next.
-8. For the sign-on URL, enter the base URL for the sample, which is by default `https://localhost:44302/`.
+7. Enter a friendly name for the application, for example "TodoListWebApp_MT", select "Web Application and/or Web API", and click the Next arrow on the lower right.
+8. For the Sign-on URL, enter the base URL for the sample, which is by default `https://localhost:44302/`.
 9. For the App ID URI, enter `https://<your_tenant_domain>/TodoListWebApp_MT`, replacing `<your_tenant_domain>` with the domain of your Azure AD tenant (either in the form `<tenant_name>.onmicrosoft.com` or your own custom domain if you registered it in Azure Active Directory).
-10. Click the Finish button on the lower right corner. Your application is now provisioned.
-11. Click on the Configure tab of your application
-12. Find "the application is multi-tenant" switch and flip it to yes. Hit the Save button from the command bar.
-12. Scroll down to the bottom of the page, to the section Permissions to other applications". On the Windows Azure Active Directory row, open the Delegated Permissions combo box and select "Enable sign-on and read users' profiles". Hit Save again.
+10. Click the Complete checkmark on the lower right corner. Your application is now provisioned and you will be returned to it's main application page.
+11. Click on the Configure tab of your application.
+12. Find "the application is multi-tenant" switch and flip it to "yes", then click the Save option at the bottom of the page.
+12. Scroll down to the "Permissions to other applications" section. On the Windows Azure Active Directory row, click on the "Delegated Permissions" combo box and note that the "Sign-on and read users profile" permission is already enabled, which is assigned by default whenever a new application is registered. This will allow our application to receive delegated permission to authenticate and read user profile data, for a given user account. The list of permissions provided here are known as permissions scopes, some of which require Administrator consent. See the [Graph API Permissions Scopes](https://msdn.microsoft.com/Library/Azure/Ad/Graph/api/graph-api-permission-scopes) article for more information.
 
 Don't close the browser yet, as we will still need to work with the portal for few more steps. 
 
 ### Step 4:  Provision a key for your app in your Azure Active Directory tenant
 
-The new customer onboarding process implemented by the sample requires the application to perform an OAuth2 request, which in turn requires to associate a key to the app in your tenant.
+The application will need to authenticate with Azure AD in order to participate in the Auth2 flow, which requires you to associate a private key with the application you registered in your tenant. In order to do this:
```

Comment:
I was thinking of the OAuth2 (yes, forgot the "O") grant flow (ie: client credentials grant flow). But I guess you could add more to mention OpenID in relation to the authentication?